src/main/java/edu/harvard/iq/dataverse/api/Dataverses.java

```diff
@@ -303,6 +308,70 @@ public Response importDataset(String jsonBody, @PathParam("identifier") String p
         }
     }
 
+    // TODO decide if I merge importddi with import just below (xml and json on same api, instead of 2 api)
+    @POST
+    @Path("{identifier}/datasets/:importddi")
+    public Response importDatasetDdi(String xml, @PathParam("identifier") String parentIdtf, @QueryParam("pid") String pidParam, @QueryParam("release") String releaseParam) throws ImportException {
```

Comment:
Shall I merge this with https://github.com/IQSS/dataverse/blob/develop/src/main/java/edu/harvard/iq/dataverse/api/Dataverses.java#L240 in order to do one api only ? I used nearly the same structure.